api/desecapi/views.py

```diff
@@ -517,21 +517,23 @@ def get(self, request, *_):
         if domain is None:
             raise NotFound('nohost')
 
-        data = {'A': self._find_ip_v4(request), 'AAAA': self._find_ip_v6(request)}
+        ipv4 = self._find_ip_v4(request)
+        ipv6 = self._find_ip_v6(request)
+
+        data = [
+            {'type': 'A', 'subname': '', 'ttl': 60, 'records': [ipv4] if ipv4 else []},
+            {'type': 'AAAA', 'subname': '', 'ttl': 60, 'records': [ipv6] if ipv6 else []},
+        ]  # FIXME why reset TTL to 60?
```

Comment:
If the RRset does not exist, we need some `ttl`. It's easiest to just set it always; don't think anyone will object (it's a dynDNS service).